app/views/ex_links/show.html.haml

```diff
@@ -0,0 +1,45 @@
+.col-md-6
+  .panel.panel-orange.margin-bottom-20
+    %table.table.table-striped
+      %thead
+        %tr
+          %th Attribute
+          %th Value
+      %tbody
+        %tr
+          %td Label
+          %td
+            = @ex_link.label
+        %tr
+          %td Url
+          %td
+            %a{ :href => @ex_link.url, :title => @ex_link.label } #{@ex_link.url}
```

Comment:
`%... \#{expression}` can be written without interpolation as `%...= expression`<br>Line is too long. [81/80]